pgdb/v1/schema.go

```diff
@@ -132,6 +142,30 @@ func readIndexes(ctx context.Context, db sqlScanner, desc Descriptor) (map[strin
 	return indexes, nil
 }
 
+func tableIsParentPartition(ctx context.Context, db sqlScanner, tableName string) (bool, error) {
+	dialect := goqu.Dialect("postgres")
+
+	qb := dialect.From("pg_class")
+	qb = qb.Select("relname")
+	qb = qb.Join(goqu.T("pg_partitioned_table"), goqu.On(goqu.I("pg_partitioned_table.partrelid").Eq(goqu.I("pg_class.oid"))))
+	qb = qb.Where(goqu.L("relname = ?", tableName))
+	query, params, err := qb.ToSQL()
+	if err != nil {
+		return false, err
+	}
+
+	rows, err := db.Query(ctx, query, params...)
```

Comment:
doesn't this need a defer rows.Close()?

https://pkg.go.dev/github.com/jackc/pgx#hdr-Query_Interface

Reply:
added the messing defers here https://github.com/ductone/protoc-gen-pgdb/pull/63/commits/53205c29b55955b9b67a74fad8e7154e00310b58